cloud/pkg/cloudhub/servers/httpserver/server.go

```diff
@@ -118,30 +155,34 @@ func edgeCoreClientCert(w http.ResponseWriter, r *http.Request) {
 		if err == jwt.ErrSignatureInvalid {
 			w.WriteHeader(http.StatusUnauthorized)
 			w.Write([]byte(fmt.Sprintf("Invalid authorization token")))
-			return
+			return false
 		}
 		w.WriteHeader(http.StatusBadRequest)
 		w.Write([]byte(fmt.Sprintf("Invalid authorization token")))
-		return
+		return false
 	}
 	if !token.Valid {
 		w.WriteHeader(http.StatusUnauthorized)
 		w.Write([]byte(fmt.Sprintf("Invalid authorization token")))
-		return
+		return false
 	}
+	return true
+}
 
+// signEdgeCert signs the CSR from EdgeCore
+func signEdgeCert(w http.ResponseWriter, r *http.Request) {
 	csrContent, err := ioutil.ReadAll(r.Body)
 	if err != nil {
-		klog.Errorf("fail to read file! error:%v", err)
+		klog.Errorf("fail to read file when signing the cert for edgenode:%s! error:%v", r.Header.Get("NodeName"), err)
```

Comment:
For the "nodename", could we use a const?

Reply:
yep. Got it